app/main.py

```diff
@@ -2,0 +1,42 @@
+from types import TracebackType
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name: str = name
+        self.health: int = health
+        self.hidden: bool = False
+        Animal.alive.append(self)
+
+    def __enter__(self) -> None:
+        pass
+
+    def __exit__(
+            self,
+            exc_type: type,
+            exc_val: Exception,
+            exc_tb: TracebackType
+    ) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
```

Comment:
In line 25, you are using concatenation to create a string. This is a code style issue. Use interpolation instead of concatenation. It is more readable and efficient. For example, you can write it like this: `f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"`.